backend/app.js

```diff
@@ -5,6 +5,7 @@ const cookieParser = require('cookie-parser');
 const logger = require('morgan');
 const session = require('express-session');
 const passport = require('./auth/passport');
+const cors = require('cors');
```

Comment:
We don't need cors , since we have the proxy set, all our axios request will be to `/api/<router>.... `
This is like a local request and it doesn't require cors
Our code shouldn't have (at any place in the frontend as well as the routes) 'http://localhost/3129'